src/azure-cli/azure/cli/command_modules/appconfig/_params.py

```diff
@@ -63,14 +64,24 @@ def load_arguments(self, _):
         c.argument('top', arg_type=top_arg_type)
         c.argument('all_', options_list=['--all'], action='store_true', help="List all items.")
         c.argument('fields', arg_type=fields_arg_type)
+        c.argument('sku', help='The sku of App Configuration', arg_type=get_enum_type(['free', 'standard']))
 
     with self.argument_context('appconfig create') as c:
         c.argument('location', options_list=['--location', '-l'], arg_type=get_location_type(self.cli_ctx), validator=get_default_location_from_resource_group)
-        c.ignore('sku')
+        c.argument('assign_identity', nargs='*', validator=validate_assigned_identity, is_preview=True, help="Accept system or user assigned identities separated by spaces. Use '[system]' to refer system assigned identity, or a resource id to refer user assigned identity. Check out help for more examples")
 
     with self.argument_context('appconfig update') as c:
         c.argument('tags', arg_type=tags_type)
 
+    with self.argument_context('appconfig identity') as c:
+        c.argument('identities', nargs='*', validator=validate_identities)
+
+    with self.argument_context('appconfig identity assign') as c:
+        c.argument('identities', help="Accept system or user assigned identities separated by spaces. Use '[system]' to refer system assigned identity, or a resource id to refer user assigned identity. Check out help for more examples")
```

Comment:
do you want to give default value for users like `az vm identity assign`, which has default value [system]?

Reply:
Good idea. Will fix

---
In reply to: [366206134](https://github.com/Azure/azure-cli/pull/11795#discussion_r366206134) [](ancestors = 366206134)

Reply:
For other parameters, we have different user scenario for identity command group. For us it is just associating some identities with this resource (App Configuration). To grant whatever access to those identities should be done through ```az role```. One more example we are following is ```az acr task identity```

---
In reply to: [366522295](https://github.com/Azure/azure-cli/pull/11795#discussion_r366522295) [](ancestors = 366522295,366206134)